apache/superset
# fix: Apply custom label colors in the Dashboard immediately

### SUMMARY
This PR fixes an issue with the custom label colors not being applied on the fly.

### BEFORE
Please check issue #11677

### AFTER

https://user-images.githubusercontent.com/60598000/132559709-01999242-a032-4a7f-b0a8-82a032f865e5.mp4

### TESTING INSTRUCTIONS
1. Open a Dashboard
2. Go to "Edit properties"
3. Got to "Advanced"
4. Change the label colors
5. Save and observe the behavior

### ADDITIONAL INFORMATION


- [x] Has associated issue: Fixes #11677
